### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_CSVPreviewData_ScreenReader:Incorrect information is announced regarding grid rows and columns.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6069]
[Check out Keros!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS1.3.1; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.9.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763
JAWS version : 2019.1810.64 ILM

**Additional Details:**
Tool Name: JAWS
MAS Reference:MAS1.3.1

**Steps to Reproduce:**
1. Install the app Azure Data Studio.
2. Connect   to database->navigate to the Data services folder->HDFS Folder->   navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button.
3. Navigate till table .
4. Start JAWS.
5. Observe that screen reader is not announcing the order of table correctly.

**Actual:**
Navigate till table using down arrow or press t to move on table directly ,screen reader announces "table with 7 of 0 columns and 13 of 0 rows column1 row 1 Test_Case_Title".
**Similar issue is also observed in following pages :**
**1.CSVSummary**: Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button->click on next button.

**Expected:**
Screen reader  should announces "table with 7 columns and 13 rows column1 row 1 Test_Case_Title ".

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
Screen reader users find it difficult to understand how many rows and columns present in table.

**MAS Reference:**
MAS1.3.1-
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[6069_[Accessibility]A11y_AzureDataStudio_CSVPreviewData_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3300382/6069_.Accessibility.A11y_AzureDataStudio_CSVPreviewData_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by udeeshagautam** [comment]
Fix should be of the pattern : https://github.com/microsoft/azuredatastudio/pull/6783

**Comment by Charles-Gagnon** [comment]
Possibly related to #6151

**Comment by Charles-Gagnon** [comment]
This is no longer announcing x of 0 rows because of the fix in https://github.com/microsoft/azuredatastudio/pull/6783, and as that issue mentions for non-virtualized content it's suggested not to use aria-rowcount/columncount since the agent can calculate the number of rows itself.
